src/OpenLoco/src/Ui.h

```diff
@@ -125,7 +125,9 @@ namespace OpenLoco::Ui
     void disposeCursors();
     void setCursor(CursorId id);
     Point32 getCursorPos();
-    void setCursorPos(int32_t x, int32_t y);
+    Point32 getCursorPosUnscaled();
+    void setCursorPos(int32_t scaledX, int32_t scaledY);
```

Comment:
I don't think we need this function, setting the cursor pos should be always screen space.

Reply:
I've simply renamed the parameters of setCursorPos from 'x' and 'y' to 'scaledX' and 'scaledY'. This should make it clearer that scaled values are expected as inputs.
The function body was left unchanged.